styles/base.css

```diff
@@ -1,5 +1,6 @@
 * {
-  @apply subpixel-antialiased;
+  @apply subpixel-antialiased
+    transition-colors;
```

Comment:
Looks good, nice touch! @RedYetiDev  I have one concern for users sensitive to this type of interaction, it may be better to use "motion-safe:transition-colors". This feature allows us to control it via the OS settings

https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion

https://tailwindcss.com/docs/hover-focus-and-other-states#prefers-reduced-motion

Reply:
Nice note, seeing as this already landed, a second PR can always change it if needed